docs/contributing/contributor-guidelines.md

```diff
@@ -1,3 +1,31 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Question.svg" /> Contributor Guidelines
 
 ---
+
+Decred development is, like the project, decentralized. Different projects and their repos can have their own own rules and norms. However, the general process for contributing across repos will be the same. Below we describe the basic process for finding work, engaging projects and the general GitHub process.
+
+## Finding Work
+
+Nearly all work occurs in public repos and is open to new contributors. A full list of repos can be found in the Decred [GitHub org](https://github.com/decred). A list of projects under active development and descriptions of them can be found on the [Projects](../../projects) page. 
+
+If new to the project, be sure to check out the user-facing [docs](https://docs.decred.org) to get an understanding of the basics of Decred. Then find a project that interests you and that you feel you have the skills to contribute to. Once you've found a project you're interested in, find an issue you'd like to work on. GitHub Issues tagged with `good first issue` are a good place to start. 
+
+## Contributing
+
+Once you've found an issue you'd like to work on, announce your plans **before** starting work. This avoids duplicate work, minimizes the chances you're working on something that might not fit with the consensus of the community, and permits discussions that can help you achieve your goal. Most projects have a dedicated chat channel for discussing project-specific work. 
+
+Some repos will also have their own specific guidelines. For instance, the dcrd repo has fairly extensive [Code Contribution Guidelines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md).
+
+## GitHub
+
+Whie different repos may have slightly different GitHub processes, below are the basic steps common to all repos.
+
+1. Fork the repo on GitHub.
+1. Clone the forked repo. 
+1. Make your changes in a local branch.
+1. When your changes are ready to be reviewed (or you just want input from other devs), push your branch to your forked repo and open a Pull Request (PR) on GitHub against the main repo. If your changes address an Issue, put `Closes #` or `Fixes #`in the PR description. This will allow GitHub to automatically close the Issue when the PR is merged. 
+1. All changes must be reviewed and receive at least one approval before they can be merged. Only team members can give official approval, but comments from other users are encouraged.
```

Comment:
To be specific, "Only team members **with write access on the repository** can give official approval"

Reply:
Thought that's what it should be. Using this wording now